src/js/player.js

```diff
@@ -450,13 +602,66 @@ class Player extends Component {
     } else if (!this.paused()) {
       this.pause();
     }
+
+    this.trigger('ended');
   }
 
   /**
    * Fired when the duration of the media resource is first known or changed
    * @event durationchange
    */
-  handleDurationChange() {
+  handleTechDurationChange() {
```

Comment:
Their is no more time display in the new skin. I don't really know what else I can do.